src/eckit/geo/grid/unstructured/UnstructuredFromGrid.cc

```diff
@@ -63,11 +61,14 @@ Grid::iterator UnstructuredFromGrid::cend() const {
 
 
 std::vector<Point> UnstructuredFromGrid::to_points() const {
-    std::vector<Point> p;
+    std::vector<Point> points;
```

Comment:
Why not compute size once and then use it in the loop?

Reply:
Because there is no default constructor for the variant type Point

Reply:
You can add `std::monostate` as the very first alternative which is well behaved.
https://en.cppreference.com/w/cpp/utility/variant/monostate

Reply:
Super you have added this, and fixed the memory problem.